README.md

```diff
@@ -14,88 +26,150 @@ To get started with the Ada Trader project follow these steps:
+      - The price from the quote associated with the Buy button _when the button was clicked_
+  - Click the Sell button for a quote:
+    - A new entry is aded to the top of the trade history
+    - The new trade entry has:
+      - The symbol from the quote associated with the Sell button
+      - An indicator that the trade was a "sell"
+      - The price from the quote associated with the Sell button _when the button was clicked_
+
+### Advice
+To keep things simple, this wave can be completed without creating any new Backbone Models, Collections, or Views. Because the trade history is "static" (the data associated with each trade will never change), we can implement this behavior entirely through events and using jQuery to update the DOM.
+
+## Wave 3 - Trading with Limit Orders
+### Trading Overview
+When traders become more advanced in their profession they begin to find that trading only with market orders can be tedious. It can be risky to click "Buy" when the quote has one price and find that your order executed after the price had increased substantially, due to other market participants. And vigilantly watching the quotes ticker to wait for an ideal "target" price to be hit might take up your whole day!
+
+For this reason advanced traders rely heavily on more advanced order types, primarily the "[limit order](#trading-vocabulary)". A limit order is similar to a market order, with two major exceptions: the limit order includes a "target price", and it remains "open" until cancelled by the trader or the target price is reached. When the current price of the stock reaches the order's target price then the order is executed (creating a new trade), and the order disappears.
+
+### Interface Details
+In the Ada Trader interface the order system can be found in the bottom panel. There are two sections to this panel: the "open orders" list and the order entry form.
+
+The right side of the bottom panel has a form for creating new limit orders. This form has a drop-down list of available stock symbols, a text input for the order's target price, and a Buy and Sell button to create the order.
+
+The left side of the bottom panel has a list of all open orders. Orders are listed with the oldest at the top. Each open order entry in the list includes the symbol being ordered, whether it is a buy or sell order, the target price, and a cancel button.
+
+### User Stories
+As a user, I can:
+  - See a list of open orders in the left side of the bottom panel, including the:
+    - Symbol for each order
+    - Buy/Sell indicator for each order
+    - [Target price](#trading-vocabulary) for each order
+    - Cancel button for each order
+  - See an order entry form in the right side of the bottom panel, including the:
+    - Symbol selection drop-down, which lists all symbols shown in the quote ticker
+    - Target price input box
+    - Buy button
+    - Sell button
+  - Click the Cancel button on each order in the open orders list
```

Comment:
Same comment re: these bullets being unnecessary

Reply:
I actually like the "buy button"/"sell button" being explicitly listed in what you can see in the order entry form since it's in the same paragraph mixed in context with other order functionality ("Click the Cancel button on each order in the open orders list" is unrelated to the order entry form)